cohere/client.py

```diff
@@ -33,12 +34,13 @@ class Client:
     def __init__(self,
                  api_key: str,
                  version: str = None,
-                 num_workers: int = 8,
+                 num_workers: int = 64,
```

Comment:
We need to be careful with making such changes. With our rate limit of 100 req/sec for trial keys a num_workers of 64 might be too much.

Reply:
These are threads not processes so they should be very cheap. (ie. doesn't really matter how many cores your CPU has)

As far as rate limiting, I think it should be the user's responsibility to handle retries or to wait between batches of calls. Maybe later we can add some convenience functions to make it easier, but in reality if users are hitting that barrier they should just grab a Production Key

Reply:
In `eve` we use [this](https://pypi.org/project/pyrate-limiter/) library to respect the APIs rate limit. 

If we decide to go for for the route of adding a convenience function, it is simple to use.

Reply:
> but in reality if users are hitting that barrier they should just grab a Production Key

Agreed. Maybe we should encourage them to do the same. Currently it is not very obvious. An error message from rate limiting that lets them know this rate limit is for the trial key and they should use a prod key might be useful.

Please feel free to resolve